src/app/nextversion/nextversion.go

```diff
@@ -100,7 +116,13 @@ func NextVersion(cCtx *cli.Context) error {
 		return err
 	}
 
+	entryCap := bump.NameToType(strings.ToLower(cCtx.String(BumpCapFlag)))
+	dependencyCap := bump.NameToType(strings.ToLower(cCtx.String(DependencyCapFlag)))
```

Comment:
It seems to me here that `NameToType` requiring the string to be lowercase is an implementation detail that could be hidden from the caller. Do you think it makes sense to move `strings.ToLower` inside of `NameToType` instead?

Reply:
It makes sense. Solved in [#c970483](https://github.com/newrelic/release-toolkit/pull/128/commits/c9704833ba2134da259a84b74334e8dc8dabf717)